src/pages/Products/PC/FilterPC.tsx

```diff
@@ -2,7 +2,7 @@ export const FilterPC = () => {
   return (
     <div className="card col-2 mt-2">
       <h5 className="card-header">Filters:</h5>
-      <ul className="list-group">
+      <ul className="list-group list-group-flush">
```

Comment:
Brak tej klasy powodował to zaokrąglenie w środku.
Już go nie ma.

![image](https://github.com/pawelNu/compstore-ui/assets/93542936/afb00861-5618-4b21-baae-83d4e2c06750)